ci-operator/step-registry/gcp/provision/private-custom-dns/gcp-provision-private-custom-dns-commands.sh

```diff
@@ -21,9 +21,6 @@ if [[ -n "${BASE_DOMAIN}" ]]; then
 fi
 
 GCP_PROJECT="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
-if [[ -n "${BASE_DOMAIN_ZONE_PROJECT}" ]]; then
```

Comment:
Why we need to remove this block ?

Reply:
I don't think the env var is necessary here, on considering that the cluster resources are always in the default GCP project (by the given cluster profile). 

FYI the env var `BASE_DOMAIN_ZONE_PROJECT` was introduced for a TP feature of GCP XPN scenarios.

Reply:
I think it is harmless to keep it, right ?

Reply:
If it is not needed for now, I am okay with dropping it as your desire.